views/js/test/runner/mocks/areaBrokerMock.js

```diff
@@ -54,31 +54,42 @@ define([
     /**
      * Builds and returns a new areaBroker with dedicated areas.
      * @param config.$brokerContainer - where to create the area broker - default to #qunit-fixture
-     * @param config.areas - A list of areas to create
+     * @param {String[]} config.areas - A list of areas to create, or...
+     * @param {Object} config.mapping - ... a list of already created areas
      * @returns {areaBroker} - Returns the new areaBroker
      */
     function areaBrokerMock(config) {
-        var mapping = {};
         var $areaBrokerDom = $('<div />').attr('id', 'area-broker-mock-' + (mockId++)).addClass('test-runner');
 
         config = config || {};
 
-        if (!config.areas) {
-            config.areas = defaultAreas;
+        if (!config.mapping) {
+            config.mapping = {};
+            if (!config.areas) {
```

Comment:
So, you must provide a mapping to be able to get areas from config. Is it wanted?